libs/clients/syslumenn/src/lib/syslumennClient.utils.ts

```diff
@@ -301,3 +307,51 @@ export const mapEstateRegistrant = (
     bankStockOrShares: syslaData.bankareikningarVerdbrefEdaHlutabref ?? false,
   }
 }
+
+// TODO: get updated types into the client
+export const mapEstateInfo = (syslaData: DanarbuUppl): EstateInfo => {
+  return {
+    assets: syslaData.eignir
+      ? syslaData.eignir
+          .filter((a) => a.tegundAngalgs === TegundAndlags.NUMBER_0)
+          .filter(
+            (a) =>
+              a?.tegundAngalgs && /^[fF]{0,1}\d{7}$/.test(a.fastanumer ?? ''),
+          )
```

Comment:
Maybe we could combine this filter
```suggestion
          .filter((a) => a.tegundAngalgs === TegundAndlags.NUMBER_0 && a?.tegundAngalgs && /^[fF]{0,1}\d{7}$/.test(a.fastanumer ?? ''))
```

Reply:
@albinagu I agree with this,do you?. If so, could you "commit suggestion" as you're the repoman?